src/main/java/net/mcreator/element/types/LivingEntity.java

```diff
@@ -150,6 +150,7 @@
 	public boolean ranged;
 	public MItemBlock rangedAttackItem;
 	public String rangedItemType;
+	public int rangedAttackDelay;

```

Comment:
You don't need a converter for this. This field is missing in JSON so it will not be set to, so you can set default value like you see for other cases here: https://github.com/MCreator/MCreator/blob/7de2294a640055ae5a7cb471ec57a5876ff27492/src/main/java/net/mcreator/element/types/LivingEntity.java#L172

Reply:
Resolved, ty!